### ISSUE TITLE
Video transcoding feature documentation

### ISSUE THREAD

**Issue by pavelsr** [mickael-kerjean/filestash#716]
This is a really nice feature, but I haven't find any documentation how to use it. 

Only enable/disable checkbox in `/admin` page, and [go code](https://github.com/mickael-kerjean/filestash/tree/master/server/plugin/plg_video_transcoder) of plugin.

How to check that this feature really works ?

How to debug it if it's not working?

Can it transcode video in live (e.g. when you are watching video that saved in non-HTML5 format) ?

Any help is appreciated.

**Comment by mickael-kerjean** [comment]
You're correct. From the new release onward, there's a couple approach video works:

1) default behavior (aka without any transcoding at all): no transcoding

2) with [the current plugin](https://github.com/mickael-kerjean/filestash/tree/master/server/plugin/plg_video_transcoder): local transcoder which does transcode to an HLS stream

3) plan to add more plugin to handle the transcoding in a separate service

4) if people are interested you could even write your own transcoder for whatever format you need but that would need to come as a plugin.
